src/ckeditor/ckeditor.component.spec.ts

```diff
@@ -4,12 +4,11 @@
  */
 
 import waitUntil from 'wait-until-promise';
-import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
I see you removed `async` module, but some tests are still async - do I understand correctly that this was never necessary here?

Reply:
Yes, because this [code is passing promises](https://angular.io/api/core/testing/TestBed#compilecomponents), so we can use that instead.